packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1379,14 +1401,51 @@ export class GarbageCollector implements IGarbageCollector {
                 );
             } else {
                 nodeStateTracker.updateTracking(currentReferenceTimestampMs);
-                if (this.tombstoneMode && nodeStateTracker.state === UnreferencedState.SweepReady) {
-                    const nodeType = this.runtime.getNodeType(nodeId);
-                    if (nodeType === GCNodeType.DataStore || nodeType === GCNodeType.Blob) {
-                        this.tombstones.push(nodeId);
-                    }
+                if (nodeStateTracker.state === UnreferencedState.SweepReady) {
+                    sweepReadyNodes.push(nodeId);
                 }
             }
         }
+
+        return sweepReadyNodes;
+    }
+
+    private validateSweepRoutes(sweepRoutes: string[], allRoutes: string[]) {
```

Comment:
Can you add comments describing what this function is doing and how? It's hard to follow otherwise.

Reply:
Will do

Reply:
Going to have the validate code in a separate PR so I'm removing this function for now.